src/example/simulation_caller.py

```diff
@@ -0,0 +1,15 @@
+import MB_dist
+
+caller(1000,30,1)
+
+def caller(number_of_particles, FPS, particle_mass):
```

Comment:
Here, you call the function before you define it, which will result in an error. I would switch the order of these lines.